src/nvim/api/private/defs.h

```diff
@@ -22,6 +22,16 @@ typedef enum {
   kErrorTypeValidation
 } ErrorType;
 
+typedef enum {
+  kMessageTypeRequest,
+  kMessageTypeResponse,
+  kMessageTypeNotification
+} MessageType;
+
+
+/// Used as the message ID of notifications.
+#define NO_RESPONSE UINT64_MAX
```

Comment:
Sorry, @Pyrohh, lua's cimport can't handle `uint64_t`, `size_t`, or the `enum` version, and I don't know how to `#if NOT_CIMPORT` it.